structured_type_test.go

```diff
@@ -1621,3 +1623,135 @@ func forAllStructureTypeFormats(dbt *DBTest, f func(t *testing.T, format string)
 		})
 	}
 }
+
+func TestNullObject(t *testing.T) {
+	skipStructuredTypesTestsOnGHActions(t)
+	runDBTest(t, func(dbt *DBTest) {
+		nullObjectQueries := []string{
+			"select null::object(v VARCHAR)",
+			"select null::object",
+		}
+		for i := 0; i < len(nullObjectQueries); i++ {
```

Comment:
Can we use `t.Run` here? See other examples in our repo :)

Reply:
Done!